ecs-cli/modules/utils/compose/convert_task_definition_test.go

```diff
@@ -1064,11 +1095,76 @@ func verifyUlimit(t *testing.T, output *ecs.Ulimit, name string, softLimit, hard
 	}
 }
 
+func TestConvertToVolumes(t *testing.T) {
+	libcomposeVolumeConfigs := map[string]*config.VolumeConfig{
+		namedVolume: nil,
+	}
+
+	expected := &volumes{
+		volumeWithHost:  make(map[string]string), // map with key:=hostSourcePath value:=VolumeName
+		volumeEmptyHost: []string{namedVolume},   // Declare one volume with an empty host
+	}
+
+	actual, err := ConvertToVolumes(libcomposeVolumeConfigs)
+
+	assert.NoError(t, err, "Unexpected error converting libcompose volume configs")
+	assert.Equal(t, expected, actual, "Named volumes should match")
+}
+
+func TestConvertToVolumesWithDriverSubfield(t *testing.T) {
```

Comment:
when the test is asserting an error, suggest we name it "TestThrowErrorOn..." or something that indicates it's testing a failure case (this goes for this and the following 2 tests)

Reply:
That's a good idea -- I figured I'd name these tests the same as other fields in case in the near future we decide to support subfields under volumes. I thought that given the test itself expects an error, it was clear enough?

Reply:
My thought was, when you run the tests all you see is the name of the test and Pass/Fail. Indicating in the test name that we're verifying an error case tells me more at the time of running the tests what the expected behavior of the package is.